scripts/install/install_macos.sh

```diff
@@ -129,19 +129,15 @@ prompt() {
   fi
 }
 
-observiq_banner()
+bindplane_banner()
 {
-  fg_cyan "           888                                        8888888 .d88888b.\\n"
-  fg_cyan "           888                                          888  d88P\" \"Y88b\\n"
-  fg_cyan "           888                                          888  888     888\\n"
-  fg_cyan "   .d88b.  88888b.  .d8888b   .d88b.  888d888 888  888  888  888     888\\n"
-  fg_cyan "  d88\"\"88b 888 \"88b 88K      d8P  Y8b 888P\"   888  888  888  888     888\\n"
-  fg_cyan "  888  888 888  888 \"Y8888b. 88888888 888     Y88  88P  888  888 Y8b 888\\n"
-  fg_cyan "  Y88..88P 888 d88P      X88 Y8b.     888      Y8bd8P   888  Y88b.Y8b88P\\n"
-  fg_cyan "   \"Y88P\"  88888P\"   88888P'  \"Y8888  888       Y88P  8888888 \"Y888888\"\\n"
-  fg_cyan "                                                                   Y8b  \\n"
-
-  reset
```

Comment:
Looks like the reset got eaten here, is that important?

Reply:
Sure is (probably). Will fix.